test/helpers/errors.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
nit) This seems like it should have 2 spaces as tabs not 4

make sure to run `yarn pretty`